### PR TITLE
GAT-5962 :: send notification email to current user

### PR DESCRIPTION
**Pull request by dnhdruk**
## Screenshots (if relevant)

Screenshot 2024-12-13 at 13 46 46
Screenshot 2024-12-13 at 13 51 52

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added appropriate unit tests
- [ ] I have created mocks for unit tests (where appropriate)
- [ ] I have added appropriate Behat tests to confirm AC (if applicable)
- [ ] I have added Swagger annotations for new endpoints (if applicable)
- [ ] I have added audit logs for new operation logic (if applicable)
- [ ] I have added new environment variables to the .env.example file (if applicable)
- [ ] I have added new environment variables to terraform repository (if applicable)

### CODE REVIEW

File: app/Console/Commands/UpdateEmailARSNotificationMessageGat5962.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        EmailTemplate::where(['identifier' => 'dar.notifymessage'])
```
Comment (BSnelling) [inline app/Console/Commands/UpdateEmailARSNotificationMessageGat5962.php:29]: This template doesn't follow the figma design for dar notifications. Why is it being updated?

**Review by JamieByrneHDRUK — APPROVED** [review]

File: app/EnquiriesManagementController/EnquiriesManagementController.php [hunk 2]
```diff
@@ -29,6 +29,12 @@ public function determineDARManagersFromTeamId(int $teamId, int $enquiryThreadId
         $enquiryThread = null;
 
         $users = [];
+        // user who open the enquiry
+        if ($currUserId) {
+            $users[] = [
+                'user' => User::where('id', $currUserId)->first()->toArray(),
```
Comment (loki-sinclair-hdruk) [inline app/EnquiriesManagementController/EnquiriesManagementController.php]: Only thing I'd ask, is it it needs the entire model?

Comment (dnhdruk) [reply app/EnquiriesManagementController/EnquiriesManagementController.php]: I sent and update

File: app/Console/Commands/UpdateEmailARSNotificationMessageGat5962.php (hunk 1 above)
Comment (dnhdruk) [reply app/Console/Commands/UpdateEmailARSNotificationMessageGat5962.php:29]: because not working

Comment (dnhdruk) [reply app/Console/Commands/UpdateEmailARSNotificationMessageGat5962.php:29]: I am in the process of updating these features, which can cover several things and one of them is .

**Review by loki-sinclair-hdruk — APPROVED** [review]

**Pull request merged**